Add checks for if certificate creation and storage was successful (#789)
aws-greengrass/aws-greengrass-lite: RELEASE_NOTES.md

# Release Notes v2.0.1 (Jan 22 2024)

This is a minor release that adds the missing features to supports default
recipe of Stream manager 2.2.0.

## New with this release

> 
> - Static build with NVIDIA jetson
> - Fix warnings from Fleet provisioning
> - Add retry and backoff algorithm to jobs listener
> - Restructured directories to have seprate directories for tests and modules
> - Add recipe variable interpolation to greengrass recipe's timeout section
> - Add support for ValidateAuthorizationToken IPC command

Reviewer: ```suggestion

- Add recipe variable interpolation to greengrass recipe's timeout section
- Add support for ValidateAuthorizationToken IPC command for stream manager
- Fix warnings from Fleet provisioning
- Add retry and backoff to jobs listener
```
